README.md

```diff
@@ -1,8 +1,8 @@
 TwigX Bundle
 =================
-![Version](https://img.shields.io/badge/version-3.0.0-blue.svg)
+[![Latest Stable Version](https://img.shields.io/packagist/v/lmc/twigx-bundle.svg)](https://packagist.org/packages/lmc/twigx-bundle)
 
-This is a Symfony bundle with Twig implementation of [Spirit Design System] components, extended with HTML-like syntax.
+Symfony bundle with Twig implementation of [Spirit Design System] components, extended with HTML-like syntax.
```

Comment:
```suggestion
Symfony bundle extending Twig template engine with [JSX]-like markup.
```

There are no Twig components of the Spirit Design System now. The description is outdated.

Also add the link to https://reactjs.org/docs/introducing-jsx.html, eg. JSX.